powerline/segments/vim/plugin/ale.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import (unicode_literals, division, absolute_import, print_function)
```

Comment:
Now it is missing modeline. LGTM, except for this and lack of spaces around some operators (`+`) issues.

Reply:
Sorry, brought back modeline and added spaces around plus signs

Reply:
Happy to squash the commits if that is your preference